doc/source/about/team.md

```diff
@@ -5,11 +5,13 @@
 GeoPandas is developed by more than [100 volunteer contributors](https://github.com/geopandas/geopandas/graphs/contributors).
 
 ## Core developers
+
 - Joris Van den Bossche - **lead maintainer** | [@jorisvandenbossche](https://github.com/jorisvandenbossche)
```

Comment:
@jorisvandenbossche you mentioned today that you don't want to formalise the role of "lead", do you want to remove it from here?